pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -237,6 +237,9 @@ const (
 	AnnotationAPIIdentityKey = "apis.kcp.dev/identity"
 )
 
+// LabelAPIBindingKindKey is a label to identify the kind of APIBinding.
+const LabelAPIBindingKindKey = "apis.kcp.dev/kind"
```

Comment:
what is it used for?

Reply:
This is used to express some intent related to this binding. See my answer [below](https://github.com/kcp-dev/kcp/pull/2329#discussion_r1022599599)